data/lark/loadData.mjs

```diff
@@ -1,109 +1,110 @@
+#!/usr/bin/env node --no-warnings
+
 import fetch from 'node-fetch';
-import fs from 'fs';
 import inquirer from 'inquirer';
 
-const urlBase = 'http://127.0.0.1:8000';
+const urlBase = 'http://django:8000';
 
 const modules = {
-	camper_pricing_logic: (await import('./camperPricingLogic.mjs')).default,
-	camper_schema: (await import('./camperSchema.mjs')).default,
-	confirmation_page_template: (await import('./confirmationPageTemplate.mjs')).default,
-	pricing: (await import('./pricing.mjs')).default,
-	registration_pricing_logic: (await import('./registrationPricingLogic.mjs')).default,
-	registration_schema: (await import('./registrationSchema.mjs')).default,
-	registration_ui_schema: (await import('./registrationUISchema.mjs')).default,
+  camper_pricing_logic: (await import('./camperPricingLogic.mjs')).default,
```

Comment:
i know this is preexisting, but i' curious -- why dynamic imports here?

Reply:
I'm actually not sure myself.  It's been so long I can't remember.  I think it has something to do with the module nature of the thing.  JS with import vs require.

Reply:
@toolgirl and I did that originally. Loading the data as modules seemed a little easier than writing and loading JSON. And I think we were just trying to be modern by using `await import` rather than `require`. :)